package.json

```diff
@@ -11,7 +11,7 @@
     "flow:coverage": "for file in src/**.js test/**.js; do echo $file; flow coverage $file; done",
     "flow:watch": "flow-watch -e js,js.flow,flowconfig --ignore lib/ --ignore node_modules/ --watch .flowconfig --watch src/ --watch test/",
     "build": "rimraf lib && babel src --out-dir lib && flow-copy-source -v src/ lib",
-    "test": "NODE_ENV=production BABEL_ENV=test nyc --reporter=lcov --reporter=text mocha $npm_package_config_mocha",
+    "test": "NODE_ENV=production BABEL_ENV=test nyc --require @babel/register --reporter=lcov --reporter=text mocha $npm_package_config_mocha",
```

Comment:
I've never needed to `--require @babel/register` for `nyc` like this, requiring it in `mocha` has always been sufficient